app/components/results/search_result_component.html.erb

```diff
@@ -14,7 +14,7 @@
         <p class="govuk-body"><%= course.qualification.humanize %></p>
         <p class="govuk-hint"><%= accredited_body %></p>
       <% else %>
-        <%= course.qualification.humanize %>
+       <%=t("find.qualification.description_with_abbreviation.#{course.qualification}").html_safe %>
```

Comment:
```suggestion
       <%=t("find.qualification.description_with_abbreviation.#{course.qualification}.html") %>
```

https://guides.rubyonrails.org/i18n.html#using-safe-html-translations

Reply:
🤯

Reply:
https://github.com/DFE-Digital/find-teacher-training/pull/1620/commits/f099056756c4f9dae7882ec5f881eec805799b49

Whats the benefit of this anyways, over `.html_safe`